hudi-common/src/main/java/org/apache/hudi/common/model/HoodieTableType.java

```diff
@@ -32,5 +36,62 @@
  * SIMPLE_LSM - A simple 2 level LSM tree.
  */
 public enum HoodieTableType {
-  COPY_ON_WRITE, MERGE_ON_READ
```

Comment:
Add common.

Reply:
hello, do you mean adding comment of `COPY_ON_WRITE` and `MERGE_ON_READ` or move original comment of these methods to the new methods?

Reply:
`COPY_ON_WRITE`, `MERGE_ON_READ`